src/parser/jobs/sge/modules/Defensives.tsx

```diff
@@ -0,0 +1,24 @@
+import {Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Defensives as CoreDefensives} from 'parser/core/modules/Defensives'
+import React from 'react'
+
+export class Defensives extends CoreDefensives {
+	protected override trackedDefensives = [
+		this.data.actions.PNEUMA,
+		this.data.actions.HOLOS,
+		this.data.actions.PANHAIMA,
+		this.data.actions.HAIMA,
+		this.data.actions.PHYSIS_II,
+		this.data.actions.ZOE,
```

Comment:
Might be worth some further discussion as to whether Zoe should be excluded since it's often bound with Pneuma.